### PR TITLE
Fix some mixups between 2D/3D in documentation

### LINKED ISSUE #7707: Page on 3D feature describes it as 2D
**Your Godot version:** 4.1

**Issue description:** In the description of the page, it describes 2d physics repeatedly, despite being a page on the 3D equivalent. It was probably copy pasted and not changed.

**URL to the documentation page:** https://docs.godotengine.org/en/stable/classes/class_physicsserver3d.html

### PR DESCRIPTION
**Pull request by AThousandShips**
Unsure if there are other cases but seems to mostly have caught all

If the changes are considered too minor compared to the impact on translation I can limit it to only cover the incorrect links, which I think should be fixed at minimum as they will throw people off 

* Fixes: godotengine/godot-docs#7707

### CODE REVIEW

File: doc/classes/CylinderShape3D.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		A 3D cylinder shape used for physics collision.
 	</brief_description>
 	<description>
-		A 2D capsule shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D].
+		A 3D cylinder shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D].
 		[b]Note:[/b] There are several known bugs with cylinder collision shapes. Using [CapsuleShape3D] or [BoxShape3D] instead is recommended.
 		[b]Performance:[/b] [CylinderShape3D] is fast to check collisions against, but it is slower than [CapsuleShape3D], [BoxShape3D], and [CylinderShape3D].
```
Comment (AThousandShips) [inline doc/classes/CylinderShape3D.xml:9]: Unsure what this was intended to mean, so didn't alter it, but assuming it should be:
```suggestion
		[b]Performance:[/b] [CylinderShape3D] is fast to check collisions against, but it is slower than [CapsuleShape3D], [BoxShape3D], and [SphereShape3D].
```

File: doc/classes/NavigationAgent3D.xml [hunk 2]
```diff
@@ -230,8 +230,8 @@
 				- [code]type[/code]: Always [constant NavigationPathQueryResult3D.PATH_SEGMENT_TYPE_LINK].
 				- [code]rid[/code]: The [RID] of the link.
 				- [code]owner[/code]: The object which manages the link (usually [NavigationLink3D]).
-				- [code]link_entry_position[/code]: If [code]owner[/code] is available and the owner is a [NavigationLink2D], it will contain the global position of the link's point the agent is entering.
-				- [code]link_exit_position[/code]: If [code]owner[/code] is available and the owner is a [NavigationLink2D], it will contain the global position of the link's point which the agent is exiting.
+				- [code]link_entry_position[/code]: If [code]owner[/code] is available and the owner is a [NavigationLink3D], it will contain the global position of the link's point the agent is entering.
```
Comment (AThousandShips) [inline doc/classes/NavigationAgent3D.xml:233]: Confirmed from the code

File: doc/classes/CylinderShape3D.xml (hunk 1 above)
Comment (kleonc) [reply doc/classes/CylinderShape3D.xml:9]: Yeah, seems like a typo. Suggested change seems correct (from these sphere is the simplest one to check against). And other docs agree with the change:
https://github.com/godotengine/godot/blob/6588a4a29af1621086feac0117d5d4d37af957fd/doc/classes/Shape3D.xml#L8
https://github.com/godotengine/godot/blob/6588a4a29af1621086feac0117d5d4d37af957fd/doc/classes/CapsuleShape3D.xml#L8

Comment (AThousandShips) [reply doc/classes/CylinderShape3D.xml:9]: Will update soon

Comment (AThousandShips) [reply doc/classes/CylinderShape3D.xml]: Done!

**Review by kleonc — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Comment by AThousandShips** [comment]
At least some of these changes needs to be cherry picked for 4.1

**Comment by AThousandShips** [comment]
It does seem to be trivial to cherry pick for 4.1, should I add the tag?

**Pull request merged**